network-mux/network-mux.cabal

```diff
@@ -55,8 +55,8 @@ library
                        -- The Windows version of network-3.1.2 is missing
                        -- functions, see
                        -- https://github.com/haskell/network/issues/484
-                       network         >=3.1.2.2 && <3.2,
-                       process         >=1.6 && <1.7,
+                       network        ^>=3.1.4,
+                       process        ^>=1.6,
```

Comment:
Do we need to update the network-mux version?

Reply:
`network-3.1.3` fixes a build failure on `Mac M1` machines, so it's good to include it. `network-3.1.4` adds `Unix Socket` support on `Windows`, not that we need it now but it's good to know.

For `process` it's just a syntactic change.